### ISSUE TITLE
Peculiar behaviors when deleting the offline map

### ISSUE THREAD

**Issue by mikewilburn** [Esri/data-collection-dotnet#78]
### On WPF:
If you delete the currently-stored offline map, the application will throw an I/O exception like this one:

![image](https://user-images.githubusercontent.com/25207711/67906996-9e584680-fb33-11e9-85f0-c4ccb2e97ffe.png)

Clicking 'OK' successfully restarts the app and reveals the offline map is indeed deleted.

However, attempting to take offline a new section of map throws this same exception. The behavior loops and I haven't figured out how to work offline yet because of it.

### On UWP:
If you delete the offline map, the application will crash right away.

The application can be re-launched just fine and reveals the offline map is indeed deleted, but on a few occasions, I've been re-prompted to sign in. Seems the crash may also force signing out the user?

**Comment by mikewilburn** [comment]
Deleting the offline map is now throwing this error:

| WPF | UWP |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/25207711/68705309-ae642300-0542-11ea-8315-adda3bbc325e.png) | ![image](https://user-images.githubusercontent.com/25207711/68705287-a3a98e00-0542-11ea-8a78-bd95e6b6bd06.png) |

Clicking 'OK' and attempting to re-log-in (ensuring I've used the correct credentials) throws the error:  
> **Map Failed to Load**: Response status code does not indicate success: 401 (Unauthorized)

Moreover, I'm still apparently logged in and have an offline map:

<details><summary>CLICK TO EXPAND</summary>
<p>

![image](https://user-images.githubusercontent.com/25207711/68705595-30ece280-0543-11ea-8e9e-91c844593697.png)

</p>
</details>

**Comment by nCastle1** [comment]
The remaining errors are a result of an authentication manager bug involving OAuth refresh tokens that is resolved at 100.7. This issue should be resolved by #84

**Comment by mikewilburn** [comment]
@nCastle1 do we need to re-test to ensure that this one was resolved when #84 was merged?

**Comment by nCastle1** [comment]
@mikewilburn Yes, I think additional testing would be in order to confirm that this is fixed.
